backend/asmlink.ml

```diff
@@ -65,13 +62,12 @@ let check_cmi_consistency file_name cmis =
     Array.iter
       (fun import ->
         let name = Import_info.name import in
-        let crco = Import_info.crc import in
+        let nonalias = Import_info.Intf.nonalias import in
```

Comment:
I'd probably call this variable `info` rather than `nonalias` since it is of type `nonalias option` rather than `nonalias` and the `None` case is perfectly valid. How it is written now it looks like this info can't be for an alias.